python/python/tests/test_async_client.py

```diff
@@ -1190,6 +1189,32 @@ async def test_sunionstore(self, redis_client: TRedisClient):
             "g",
         }
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_sdiff(self, redis_client: TRedisClient):
+        key1 = f"{{testKey}}:1-{get_random_string(10)}"
+        key2 = f"{{testKey}}:2-{get_random_string(10)}"
+        string_key = f"{{testKey}}:4-{get_random_string(10)}"
+        non_existing_key = f"{{testKey}}:5-{get_random_string(10)}"
+
+        assert await redis_client.sadd(key1, ["a", "b", "c"]) == 3
+        assert await redis_client.sadd(key2, ["c", "d", "e"]) == 3
+
+        assert await redis_client.sdiff([key1, key2]) == {"a", "b"}
+        assert await redis_client.sdiff([key2, key1]) == {"d", "e"}
+
+        assert await redis_client.sdiff([key1, non_existing_key]) == {"a", "b", "c"}
```

Comment:
compare result with set please

Reply:
Can you clarify? `{"a", "b", "c"}` is a set, so I think I already am comparing the result with a set

Reply:
My mistake, please disregard